core/services/chainlink/config_general.go

```diff
@@ -60,6 +60,9 @@ type generalConfig struct {
 //
 // See ParseTOML to initilialize Config and Secrets from TOML.
 type GeneralConfigOpts struct {
+	ConfigStrings []string
+	SecretsString string
```

Comment:
Do `Config` and `Secrets` still need to be fields on this struct? Or could they just be local vars scoped to the `New()` method?

Reply:
They are still needed: some tests pass in Config/Secrets as structs rather than as strings.